docs/use_cases/retrieval_augmented_generation_eval_qdrant_ragas.md

```diff
@@ -0,0 +1,618 @@
+  Avoid questions such as 'in the given passage or document chunk' kind of questions. Ensure the question is about the concept the document chunk is about.
+  Provide a complete , detailed and accurate answer to the question. Make sure that the answer addresses the question directly and comprehensively, drawing from the information provided in the document chunk.
+  Use technical terminology appropriately and maintain clarity throughout the response.
+  Based on this Context : {context}
+  """
+        
+  response = client.chat.completions.create(
+        model = model,
+        messages = [ 
+        {'role': 'user', 'content': prompt}
+          ],
+        temperature=0.5,
+        max_tokens=200,
+        n=1,
+        stop=None
+    )
+  return response.choices[0].message.content
+```
+
+Using the subroutine above to generate question and answer/ground-truth pairs as :
+
+```python
+# Generate question-answer pairs for the given chunk
+
+context = "What is Qdrant? [Qdrant](http://qdrant.tech) is a vector similarity search engine that provides a \
+production-ready service with a convenient API to store, search, and manage points (i.e. vectors) with an additional payload.\
+You can think of the payloads as additional pieces of information that can help you hone in on your search and also receive \
+useful information that you can give to your users. You can get started using Qdrant with the Python `qdrant-client`, \
+by pulling the latest docker image of `qdrant` and connecting to it locally, or by trying out [Qdrant's Cloud](https://cloud.qdrant.io/) \
+free tier option until you are ready to make the full switch. With that out of the way, let's talk about what are vector databases."
+
+question_answer_pair = generate_question_answer(context)
+print(question_answer_pair)
+
+## Outputs 
+# Q . What is Qdrant and how can it be used in vector similarity search?
+# A . Qdrant is a vector similarity search engine that offers a production-ready service with an API for storing, searching, and managing points (vectors) along with additional payload information. The payloads can provide extra details to refine searches and offer valuable information to users. Qdrant can be utilized through the Python `qdrant-client`, by downloading the latest docker image of `qdrant` and connecting to it locally, or by exploring the free tier option of [Qdrant's Cloud](https://cloud.qdrant.io/) before transitioning to the full version.
+```
+
+We saw another way to generate questions along with answers/ground-truth using OpenAI prompts this time.
+
+However , as we are discussing about RAGAS , it comes with an easier way to generate Question-Context-Ground_Truth set and rather a complete baseline evaluation dataset [utility](https://docs.ragas.io/en/latest/getstarted/testset_generation.html) in RAGAS , using your dataset in just a couple of lines of code as below.
```

Comment:
maybe a 3rd title here to signal this is a 3rd way? (the suggested one)

Reply:
Added !